src/Refactoring-Core/ReVariableNameCondition.class.st

```diff
@@ -21,7 +21,8 @@ ReVariableNameCondition class >> name: aString [
 
 { #category : 'utilities' }
 ReVariableNameCondition class >> reservedNames [
-	^#('self' 'true' 'false' 'nil' 'thisContext' 'super')
+
+	^ PseudoVariable lookupDictionary keys
```

Comment:
IMO this method should be removed and users should reference `Symbol reservedLiterals`.